src/main/types/generalInfo.ts

```diff
@@ -68,6 +68,10 @@ export class GeneralInfo {
         return this._artifactId + ':' + this._version;
     }
 
+    public getFullComponentId(): string {
+        return this._pkgType + '//' + this._artifactId + ':' + this._version;
+    }
+
```

Comment:
Looks not in use and it is also may be wrong - the package type does not equal to the package prefix
```suggestion
```

Reply:
left over from testing. removed